src/account/saga.test.ts

```diff
@@ -1,4 +1,4 @@
-import * as DEK from '@celo/utils/lib/dataEncryptionKey'
+import * as DEK from '@celo/cryptographic-utils/lib/dataEncryptionKey'
```

Comment:
same as above, maybe these could be in a followup.

Reply:
I can do that in a followup PR